### ISSUE TITLE
Consider contributing a ZeroNet client to Beaker

### ISSUE THREAD

**Issue by OliverCole** [HelloZeroNet/ZeroNet#530]
> Beaker is a browser for IPFS and Dat. It's licensed MIT, and supports plugins for adding new protocols and Web APIs.

https://github.com/pfrazee/beaker

**Comment by pfrazee** [comment]
I'm available to help out any plugin devs, and we have some good docs. (I may build a zeronet plugin myself; we'll see what my schedule looks like.)

I started `#beakerbrowser` in Freenode; you can find me there

**Comment by obv-mikhail** [comment]
Using a browser for the single purpose of making URLs look pretty seems stupid to me.

**Comment by dmp1ce** [comment]
I don't think it is stupid if it stop cross site scripting attacks! Does beaker prevent scripts from running on other ZeroNet pages? I'll take a look.

**Comment by Richard87** [comment]
I second that, security and stopping XSS is important, and maybe this browser would make it way easier to spread/setup a local ZeroNet innstalation if its built into the plugin?

**Comment by HelloZeroNet** [comment]
The XSS is not really an issue on ZeroNet because of iframe sandboxing and we have no POST/GET api, but it would be good to add it as a plugin.
Also worth checking: https://github.com/kustomzone/Fuzium
